install.js

```diff
@@ -104,7 +105,7 @@ function exit(code) {
 function findSuitableTempDirectory() {
   var now = Date.now()
   var candidateTmpDirs = [
-    process.env.TMPDIR || process.env.TEMP || process.env.npm_config_tmp,
+    os.tmpdir(),
```

Comment:
I believe this would break node < v4 support.
Also I'd readd `process.env.npm_config_tmp`. Otherwise that would be a breaking change.
```js
var candidateTmpDirs = [
  process.env.npm_config_tmp,
  '/tmp',
  path.join(process.cwd(), 'tmp')
]
```